services/order.py

```diff
@@ -0,0 +1,46 @@
+from django.utils import timezone
+from db.models import User, Ticket, Order, MovieSession
+from typing import Optional
+from django.db import transaction
+from datetime import datetime
+
+
+def create_order(*, username, tickets, date=None):
```

Comment:
Avoid using mutable default arguments like lists. This can lead to unexpected behavior. Use `None` as the default value and initialize the list inside the function if needed.